app/views/layouts/application.html.erb

```diff
@@ -9,9 +9,17 @@
     <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon">
     <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
 
-    <!-- Webpack Assets -->
-    <%= javascript_pack_tag 'webpack-bundle' %>
-    <%= stylesheet_pack_tag 'webpack-bundle' %>
+    <!-- Webpack Assets START -->
+    <%= javascript_pack_tag 'shared_bundle' %>
+    <%= stylesheet_pack_tag 'shared_bundle' %>
+
+    <!-- TODO: Need a way to load this bundle only on Moments pages -->
+    <%= javascript_pack_tag 'moments-app_bundle' %>
```

Comment:
my thoughts here is that once we move the entirety of the app to React, we can just inject one bundle. This feels like a  transition code atm.